jobs/list.ts

```diff
@@ -62,4 +64,6 @@ export const regularJobs: ScheduledJob[] = [
     { cronTime: '00 00 10 * * *', name: 'flagInactiveConversationsAsReadonly' },
     // Every night, trigger actions for courses that ended yesterday
     { cronTime: '00 00 10 * * *', name: 'notificationsEndedYesterday' },
+    // Every six hours look for unreachable achievements and delete them
+    { cronTime: '00 00 */6 * * *', name: 'deleteUnreachableAchievements' },
```

Comment:
(as an alternative, we could also hook it into notificationsEndedYesterday - there we anyways loop courses that ended. But a separate job is also fine)

Reply:
Interesting idea 👍 
I would keep it separate tho, for observability reasons.